src/main/resources/ath-container/Dockerfile

```diff
@@ -20,10 +20,15 @@ RUN apt-get clean && \
         vnc4server
 
 # All we need is a statically linked client library - no need to install daemon deps: https://get.docker.com/builds/
-RUN curl -fsSLO https://get.docker.com/builds/Linux/x86_64/docker-17.04.0-ce.tgz && \
-    tar --strip-components=1 -xvzf docker-17.04.0-ce.tgz -C /usr/local/bin
+RUN curl -fsSLO https://download.docker.com/linux/static/stable/x86_64/docker-17.03.2-ce.tgz && \
+    tar --strip-components=1 -xvzf docker-17.03.2-ce.tgz -C /usr/local/bin
+
 ENV SHARED_DOCKER_SERVICE true
 
+# Selenium needs a geckodriver in order to work properly
```

Comment:
In general or for firefox only? Why it is not installed through `apt-get`? I presume there are reasons but they are not obvious to me - so please document them.

Reply:
I think it is necessary only for Firefox.
I am not sure if it is available from apt-get - it was from rtyler, but it is worthy to check.

Reply:
It is not possible from apt-get.